File: src/main/java/org/example/InitializeUserList.java
package org.example;

public class InitializeUserList {

    UserList userList = new UserList();

    public UserList initializeUserList(){
        userList.addUser(new User("alice", "pass123"));
        userList.addUser(new User("bob", "pass456"));
        userList.addUser(new User("charlie ", "pass789"));
        return userList;
    }
}

File: InitializeUserList.js
const User = require('./User');
const UserList = require('./UserList');

class InitializeUserList {
    constructor() {
        this.userList = new UserList();
    }

    initializeUserList() {
        this.userList.addUser(new User("alice", "pass123"));
        this.userList.addUser(new User("bob", "pass456"));
        this.userList.addUser(new User("charlie", "pass789"));
        return this.userList;
    }
}

module.exports = InitializeUserList;